internal/services/controller/controller.go

```diff
@@ -471,6 +477,24 @@ func (c *Controller) informerIsAllowedToAccessResource(ctx context.Context, name
 func (c *Controller) Start(done <-chan struct{}) {
 	c.informerFactory.Start(done)
 }
+func processApiResourcesError(log logrus.FieldLogger, err error) map[schema.GroupVersion]bool {
```

Comment:
note: The name is a little bit implicit here.

WIthout reading the method it's not possible for me to understand what it does.